tests/nested/core/save-data/task.yaml

```diff
@@ -12,7 +12,7 @@ execute: |
     tests.nested exec "sudo snap wait system seed.loaded"
 
     echo "Ensuring the presence of ubuntu-save"
-    tests.nested exec "lsblk -f -m -l | awk '{ print \$3 }' | grep ubuntu-save"
+    tests.nested exec "lsblk -oLABEL" | MATCH '^ubuntu-save$'
```

Comment:
`blkid -L ubuntu-save` might be a bit simpler. Though it is fine like that.

Reply:
Nice one, I updated the PR! Thank you!